components/LogItem.vue

```diff
@@ -21,33 +17,38 @@ const logSorted = computed(() => {
 function objtypeFull(objtype: ObjType) {
   return objTypeFull(objtype)
 }
+
+function uniqHistoryIds(log: Log) {
+  return _.uniq(
+    _.compact([log.base, log.change])
+      .map((object) => ({ objtype: object.objtype, id: object.id })),
+    (object) => `${object.objtype}${object.id}`,
+  )
+}
 </script>
 
 <template>
   <el-card shadow="never">
     <template #header>
       <div class="card-header">
-        <span>
-          <a
-            :href="`https://www.openstreetmap.org/${objtypeFull(log.objtype)}/${log.id}/history`"
-            target="_blank"
-          >
-            {{ log.objtype }}{{ log.id }}
-          </a>
-          -
-          {{ log.base?.tags.name || log.change.tags.name }}
-        </span>
+        <template v-for="object in uniqHistoryIds(log)">
+          <span v-if="object" :key="`${object.objtype}${object.id}`">
+            <a
+              :href="`https://www.openstreetmap.org/${objtypeFull(object.objtype)}/${object.id}/history`"
+              target="_blank"
+            >
+              {{ object.objtype }}{{ object.id }}
+            </a>
+            -
+            {{ log.base?.tags.name || log.change.tags.name }}
+          </span>
+        </template>
```

Comment:
This part doesn't seems to work, we have `undefined` values from `uniqHistoryIds` function, which leads to buggy display.

![image](https://github.com/user-attachments/assets/832ecaf0-630f-4cc1-8614-917df5a577d1)

Reply:
Sorry I was on PROD API...